plugins/flytekit-data-fsspec/setup.py

```diff
@@ -4,7 +4,7 @@
 
 microlib_name = f"flytekitplugins-data-{PLUGIN_NAME}"
 
-plugin_requires = ["flytekit>=0.21.3,<1.0.0", "fsspec>=2021.7.0"]
+plugin_requires = ["flytekit>=0.21.3,<1.0.0", "fsspec>=2021.7.0", "botocore>=1.23.24"]
```

Comment:
can we just use `>=1.0?`?  or at least lower it a bit?  the exception probably has been there for a while.  1.23.24 is the latest/very recent version.  some people might not want to upgrade.